lib/spack/spack/package.py

```diff
@@ -914,9 +913,12 @@ def configure_args_path(self):
     @property
     def install_configure_args_path(self):
         """Return the configure args file path on successful installation."""
-        install_path = spack.store.layout.metadata_path(self.spec)
+        return os.path.join(self.metadata_dir, _spack_configure_argsfile)
 
-        return os.path.join(install_path, _spack_configure_argsfile)
+    @property
+    def install_test_root(self):
+        """Return the install test root directory."""
+        return self.metadata_dir
```

Comment:
It looks like this is putting all of the files directly into `$prefix/.spack`, can we add a `test` subdirectory here (now that the entries can be individual files)? It looks like everything is well-factored enough we could just do

```suggestion
        return os.path.join(self.metadata_dir, 'test')
```